pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -181,7 +181,7 @@ func (alloc *Action) allocateResourcesForTasks(tasks *util.PriorityQueue, job *a
 		klog.V(3).Infof("There are <%d> nodes for Job <%v/%v>", len(ssn.Nodes), job.Namespace, job.Name)
 
 		if err := ssn.PrePredicateFn(task); err != nil {
-			klog.V(3).Infof("PrePredicate for task %s/%s failed for: %v", task.Namespace, task.Name, err)
+			klog.V(4).Infof("PrePredicate for task %s/%s failed for: %v", task.Namespace, task.Name, err)
```

Comment:
log levels are different in this file, we'd better unify them to the same, but whether to 3 or 4 need to be discussed, so does other file like recalim: )

Reply:
We'd better keep log level 3, because it will stop allocating. If use level 4, user can not easy to find out why job is not schedulable. 

same to the others.

Reply:
That's true, why change to level 4? @googs1025

Reply:
my bad. aready rolled back! thank for reminding this point.